src/components/menu.svelte

```diff
@@ -50,7 +50,7 @@
         startDate={w.startDate}
         endDate={w.endDate}
         summary={w.summary}
-        highlights={w.highlights}
+        newHighlights={w.newHighlights}
```

Comment:
Need to replace `newHighlights` with just `highlights`, now that the old highlight property is no longer necessary.